### PR TITLE
Bulk fiscal event push (Array of fiscal event)

### PR DESCRIPTION
**Pull request by pintu-eGov**

### CODE REVIEW

File: domain-services/fiscal-event-service/src/main/java/org/egov/service/FiscalEventEnrichmentService.java [hunk 1]
```diff
@@ -26,6 +26,37 @@
 
     public void enrichFiscalEventPushPost(FiscalEventRequest fiscalEventRequest) {
         RequestHeader requestHeader = fiscalEventRequest.getRequestHeader();
+<<<<<<< HEAD
```
Comment (rushang7-eGov) [inline domain-services/fiscal-event-service/src/main/java/org/egov/service/FiscalEventEnrichmentService.java:29]: Why is there <<< HEAD here? Does this code run properly with this?

File: domain-services/fiscal-event-service/src/main/java/org/egov/service/FiscalEventEnrichmentService.java [hunk 2]
```diff
@@ -53,5 +84,6 @@ public void enrichFiscalEventPushPost(FiscalEventRequest fiscalEventRequest) {
             fiscalEvent.setIngestionTime(System.currentTimeMillis());
         }
 
+>>>>>>> f070c61465b100be594b1916109e464860bcc3cb
```
Comment (rushang7-eGov) [inline domain-services/fiscal-event-service/src/main/java/org/egov/service/FiscalEventEnrichmentService.java:87]: ??

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: domain-services/fiscal-event-service/src/main/java/org/egov/service/FiscalEventEnrichmentService.java (hunk 2 above)
Comment (pintu-eGov) [reply domain-services/fiscal-event-service/src/main/java/org/egov/service/FiscalEventEnrichmentService.java]: resolved.

File: domain-services/fiscal-event-service/src/main/java/org/egov/service/FiscalEventEnrichmentService.java (hunk 1 above)
Comment (pintu-eGov) [reply domain-services/fiscal-event-service/src/main/java/org/egov/service/FiscalEventEnrichmentService.java]: resolved

File: domain-services/ifix-master-data-service/src/test/java/org/egov/validator/ChartOfAccountValidatorTest.java [hunk 3]
```diff
@@ -303,103 +338,167 @@ void testValidateSearchPostWithSubMajorHeadWithLength() {
...
 
     @Test
     void testValidateSearchPostWithMajorHeadLength() {
         COASearchCriteria coaSearchCriteria = coaSearchRequest.getCriteria();
         coaSearchCriteria.setMajorHead("Major Header");
         RequestHeader requestHeader = mock(RequestHeader.class);
+<<<<<<< HEAD
+        when(requestHeader.getUserInfo()).thenReturn(new UserInfo());
+=======
+>>>>>>> f070c61465b100be594b1916109e464860bcc3cb
```
Comment (rushang7-eGov) [inline domain-services/ifix-master-data-service/src/test/java/org/egov/validator/ChartOfAccountValidatorTest.java:365]: ??

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: domain-services/ifix-master-data-service/src/test/java/org/egov/validator/ChartOfAccountValidatorTest.java (hunk 3 above)
Comment (pintu-eGov) [reply domain-services/ifix-master-data-service/src/test/java/org/egov/validator/ChartOfAccountValidatorTest.java]: resolved.

**Pull request closed without merging**